cypress/e2e/content/meta.spec.js

```diff
@@ -42,40 +65,47 @@ describe("Content Meta", () => {
       .type("new Meta Title");
 
     cy.get("[data-cy=metaDescription]")
-      .find("textarea")
+      .find("textarea:eq(0)")
       .click()
       .clear()
       .type("new Meta Description");
 
     cy.get("[data-cy=metaKeywords]")
-      .find("textarea")
+      .find("textarea:eq(0)")
       .click()
       .clear()
       .type("key, words, here");
 
     cy.get("[data-cy=sitemapPriority] >  div").click();
-    cy.get('[data-value="-1"]').click();
+    cy.get('[data-option-index="0"]').click();
 
     cy.get("[data-cy=canonicalTag] >  div").click();
-    cy.get('[data-value="1"]').last().click();
+    cy.get('[data-option-index="1"]').last().click();
 
     cy.get("[data-cy=canonicalTag] >  div").click();
-    cy.get('[data-value="0"]').last().click();
+    cy.get('[data-option-index="0"]').last().click();
+
+    cy.get("#parentZUID input").clear().wait(500).type("e2e");
```

Comment:
We should ideally be using data-cy attribute selectors instead of IDs to keep things robust and follow [cypress best practices](https://docs.cypress.io/app/core-concepts/best-practices#Selecting-Elements).